schemer/__init__.py

```diff
@@ -129,6 +125,8 @@ def _verify_type(self, spec, path):
         elif not isinstance(field_type, type) and not isinstance(field_type, types.FunctionType):
             raise SchemaFormatException("Unsupported field type at {}. Type must be a type, a function, an Array or another Schema", path)
 
+    def _valid_schema_default(self, value):
+        return value == {}
```

Comment:
I don't think we want to necessarily restrict people to empty dict. I'd say we just check that the value is of type `dict`.